File: Exercises_C++/1013.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){

    int x;
    int y;
    int z;

    cin >> x >> y >> z;

    int maior1;
    maior1 = (x + y + abs(x-y)) / 2;

    int maior_total;
    maior_total = (maior1 + z + abs(maior1 - z)) / 2;

    cout << maior_total << " eh o maior" << endl;

}

File: Bee-Crowd/Exercises_Python/1013.py
a, b, c = input().split()
a = int(a)
b = int(b)
c = int(c)

m_two = ((a + b + abs(a - b))) / 2

maior = ((m_two + c + abs(m_two - c))) / 2

print(int(maior), 'eh o maior')